### PR TITLE
3.0 - Dedicated article for rights on reference entities

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## The edit right
+The edit right means add and edit rights.  
+If a user belonging to a user group has right to `EDIT` a reference entity, he will be able to:
+*   Edit the reference entities labels for a locale
+*   Edit the reference entities attributes labels for a locale
+*   Edit the reference entity records values for a locale if the attribute is localizable
+
+## The view right
+If a user belonging to a user group has right to `VIEW` on a locale, he will only be able to display the reference entity data on this locale (properties, attributes, records) and that's all.  
+He will have no possibility to edit it.
```
Comment (morganen) [inline content/md/permissions-settings/access-rights-on-entities.md]: He won’t be able to edit or delete it.

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Set rights to a locale
```
Comment (morganen) [inline content/md/permissions-settings/access-rights-on-entities.md]: On a locale

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+themes: permissions-settings, manage-your-reference-entities
+title: Set rights on your **reference entities**
+popular: false
+ee-only: true
+related: what-about-reference-entities, manage-your-reference-entities, build-your-user-groups
+---
+
+# The basics
+
+Once you've [created your 1st reference entity](manage-your-reference-entities.html) and [created your user groups](build-your-user-groups.html), you can begin to define for each of these users groups if they will be able to display/edit this reference entity and its data.
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/permissions-settings/access-rights-on-entities.md]: ok  done

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+For reference entities, we have 2 levels of access: the view right and the edit right.
+
+::: warning
+It’s not possible to hide a reference entity.
+:::
+
+## The edit right
+The edit right means add, edit and delete rights.  
+If a user belonging to a user group has right to `EDIT` a reference entity, he will be able to:
+*   Edit the reference entities properties
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/permissions-settings/access-rights-on-entities.md]: ok done

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+*   Add, edit and delete the reference entities attributes
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/permissions-settings/access-rights-on-entities.md]: ah yes bien vu

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+*   Add, edit and delete a reference entity record
+*   Delete the reference entity
+
+:::info
+If you would like that only some users can edit the attributes of a reference entity or delete a record, you can also manage accesses on the actions according to [the user roles](build-your-user-roles.html), please refer to [this article](manage-the-interface-and-actions-accesses.html#rights-on-reference-entities-ee-only).
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/permissions-settings/access-rights-on-entities.md]: done

File: content/md/permissions-settings/access-rights-on-entities.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+:::
+
+## The view right
+If a user belonging to a user group has right to `VIEW` a reference entity, he will only be able to display the reference entity data (properties, attributes, records) and that's all.  
+He will have no possibility to edit or delete it.
+
+## Set rights to a reference entity
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/permissions-settings/access-rights-on-entities.md]: ok

**Comment by morganen** [comment]
GTM

**Pull request merged**